File: CS/E3465/WPF/WpfReportPreviewUserControl.cs
using System.Windows.Forms;
using DevExpress.XtraReports;

namespace E3465.WPF {
    public partial class WpfReportPreviewUserControl : UserControl {
        public IReport Report {
            get { return wpfReportPreviewControl1.Report; }
            set { wpfReportPreviewControl1.Report = value; }
        }
        public WpfReportPreviewUserControl() {
            InitializeComponent();
        }
    }
}

File: VB/E3465/WPF/WpfReportPreviewUserControl.vb
Imports Microsoft.VisualBasic
Imports System.Windows.Forms
Imports DevExpress.XtraReports

Namespace E3465.WPF
	Partial Public Class WpfReportPreviewUserControl
		Inherits UserControl
		Public Property Report() As IReport
			Get
				Return wpfReportPreviewControl1.Report
			End Get
			Set(ByVal value As IReport)
				wpfReportPreviewControl1.Report = value
			End Set
		End Property
		Public Sub New()
			InitializeComponent()
		End Sub
	End Class
End Namespace
